docs/libcudacxx/standard_api/numerics_library/linalg.rst

```diff
@@ -0,0 +1,31 @@
+.. _libcudacxx-standard-api-numerics-linalg:
+
+``<cuda/std/linalg>``
+============================================
+
+Provided functionalities
+------------------------
+
+- ``scaled()`` `std::linalg::scaled <https://en.cppreference.com/w/cpp/numeric/linalg/scaled>`_
```

Comment:
Not sure if we want to add our own short documentation for these (sound taxing), but cppref has no information whatsoever yet, so a new user would have to dig on their own to find out what they are doing.